### PR TITLE
April 2024 PSU + Fixes

### PR DESCRIPTION
**Pull request by Luigi96**
This PR contains:

- April PSU 2024 data for Adoptium Marketplace
- Quick fix in versioning schema for previous releases
- Modification of signing script to be supported in different platforms

### CODE REVIEW

**Review by d3r3kk — APPROVED** [review]

File: scripts/sign.sh [hunk 1]
```diff
@@ -45,5 +45,5 @@ for json in $(ls ${content_dir}/*.json); do
     openssl dgst -sha256 -verify "$public_key_file" -signature ${json}.sig $json
 
     # Dump signature to base64 encoded file for human readability
-    cat ${json}.sig | base64 -w 0 > ${json}.sha256.sign
+    cat ${json}.sig | base64 | tr -d '\n' > ${json}.sha256.sign
```
Comment (jmjaffe37) [inline scripts/sign.sh:48]: Just wondering, what is the benefit of this change in the signing script over the previous code?

Comment (jmjaffe37) [reply scripts/sign.sh:48]: Nevermind, I see that this helps support other platforms (from your description above)

**Comment by jmjaffe37** [comment]
This review looks good to me, but I think it needs to pass the review-bot checks. With a quick view through the logs, it seems that either one of the links in the JSONs is incorrect, or one of the signatures is incorrect

**Comment by Luigi96** [comment]
> This review looks good to me, but I think it needs to pass the review-bot checks. With a quick view through the logs, it seems that either one of the links in the JSONs is incorrect, or one of the signatures is incorrect

It seems that the issue is on retrieving maven dependencies. I will talk with @gdams to double check what is going on.

**Review by jmjaffe37 — APPROVED** [review]

**Pull request merged**